gitea/repo_tree.go

```diff
@@ -20,10 +20,13 @@ type GitEntry struct {
 
 // GitTreeResponse returns a git tree
 type GitTreeResponse struct {
-	SHA       string     `json:"sha"`
-	URL       string     `json:"url"`
-	Entries   []GitEntry `json:"tree"`
-	Truncated bool       `json:"truncated"`
+	SHA        string     `json:"sha"`
+	URL        string     `json:"url"`
+	Entries    []GitEntry `json:"tree"`
+	Truncated  bool       `json:"truncated"`
+	Page       int        `json:"page"`
+	TotalCount int        `json:"total_count"`
+	TotalPages int        `json:"total_pages"`
```

Comment:
why would you need TotalPages? You can calculate that  from total count and page size (that requester already should know)

Reply:
@lafriks Ok, removed TotalPages